API.md

```diff
@@ -309,7 +309,7 @@ annotations: [{
   title: 'Important',
   subtitle: 'Neat, this is a custom annotation image',
   annotationImage: {
-    source: { uri: 'https://cldup.com/7NLZklp8zS.png' },
+    source: { uri: 'file:///var/mobile/Containers/Data/Application/8821CE3F-C0DA-4B86-B4F7-85DE1E158943/Documents/custom-annotation.png' },
```

Comment:
Perhaps we could say how you get this path in React Native, i.e. via `react-native-fs` or `react-native-fetch-blob`s File access APIs